### DISCUSSION TITLE
pushing-nuget-packages-built-in-docker-by-running-the-container/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#35] [Announcements]
# pushing-nuget-packages-built-in-docker-by-running-the-container/

In this post I show how to create a Dockerfile for building your NuGet packages which you can then run as a container to push them to a NuGet feed.

https://andrewlock.net/pushing-nuget-packages-built-in-docker-by-running-the-container/

**Comment by macfma01** [comment]
Fantastic! Exactly the details I was looking for to get this task done for my project.

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at August 05 2018, 10:43, in reply to [Tom Davis](https://disqus.com/by/disqus_4mG9cO9Y25/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/35#discussioncomment-6045970)</em>

---

<p>I would build those as 2 different images. The API should use multi-stage builds, and publish the API to one of the aspnet runtime images. I would probably just build the NuGet packages in the aspnet SDK image, and if all the tests pass etc, push directly from the SDK builder image.</p>

[a comment by an automated account is not shown]
